admin/includes/javascript_loader.php

```diff
@@ -29,6 +29,23 @@
     }));
   });
 </script>
+<?php
+$searchBoxScriptArray = [
+    'specials',
+    'coupon_admin',
+    'reviews',
+    'featured',
+    'customers',
+    'category_product_listing',
+    'downloads_manager',
+];
+$searchBoxJs = 'includes/javascript/searchBox.js';
```

Comment:
Perhaps a notification here to enable the search functionality to be included on other pages, too?

Reply:
one of us is confused.  this functionality is dependent on the following module:

admin/includes/modules/search_box.php

the scripts listed above are the only scripts that make use of that module.  if a new (or existing) script makes use of the search_box module, then by all means, said script should be included in the array.  

if a new script not part of the base ZC makes use of the search_box module, there already is a mechanism for including `js` files, and an include in said file to the `searchBox.js` would be appropriate.

or am i confused?

Reply:
It just seemed (to me, at least) that providing a means for a plugin to hook into the already-existing code to prevent duplication would be desired.